<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RECarClient</class>
 <widget class="QDialog" name="RECarClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Costumer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReject">
        <property name="text">
         <string>Reject</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>92</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="EQLineEdit" name="leCode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="PrimaryKey" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Card code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Count of visit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="EQLineEdit" name="leVisits"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Balance</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="EQLineEdit" name="leFiscal">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="Field" stdset="0">
         <string>f_fiscal</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>ՀԴՄ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="EQLineEdit" name="leBalance">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Գին</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="EQLineEdit" name="lePrice"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="EQLineEdit" name="leCardcode">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="Field" stdset="0">
      <string>f_code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="EQLineEdit" name="leInfo">
     <property name="Field" stdset="0">
      <string>f_info</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
